apps/dashboard/app/views/layouts/application.html.erb

```diff
@@ -6,6 +6,9 @@
 
   <%= javascript_include_tag 'application', nonce: true %>
   <%= stylesheet_link_tag 'application', nonce: true, media: 'all' %>
```

Comment:
```suggestion
  <%= stylesheet_link_tag 'application', nonce: content_security_policy_nonce, media: 'all' %>
```

While I'm looking at this, and I know you didn't cause this, but this is being rendered as `nonce="true"` in HTML which is wrong. Apparently the `stylesheet_link_tag` isn't the same as `javascript_include_tag`.

Reply:
Will update